editor/src/clj/editor/material.clj

```diff
@@ -78,7 +78,7 @@
 (defn- build-target-attributes [attribute-infos]
   (mapv graphics/attribute-info->build-target-attribute attribute-infos))
 
-(g/defnk produce-base-pb-msg [name vertex-program fragment-program vertex-constants fragment-constants samplers tags vertex-space max-page-count]
+(g/defnk produce-base-pb-msg [name vertex-program fragment-program vertex-constants fragment-constants ^:raw samplers tags vertex-space max-page-count]
```

Comment:
Use the `^:raw` property value that may contain `:filter-mode-mag-default` or `:filter-mode-min-default` values when building, editing and saving.

The protobuf form editor is a bit inflexible in how it can present data for editing, so using the `^:raw` tag was the easiest way to separate uses of the property value and the newly introduced output that resolve the sampler filter modes to the the `game.project` setting.

The alternative would be to rename the output, but it is used in a bunch of places, including in the editor extensions.